tools/postinstall-android.js

```diff
@@ -100,7 +100,7 @@ if (data.indexOf("SalesforceHybrid") < 0)
     var oldGradleToolsVersion = "com.android.tools.build:gradle:2.2.3";
     var newGradleToolsVersion = "com.android.tools.build:gradle:2.3.2";
     replaceTextInFile(path.join(appProjectRoot, 'build.gradle'), oldGradleToolsVersion, newGradleToolsVersion);
-    var newLibDep = "compile \"com.google.android.gms:play-services-gcm:7.5.0\"\ncompile project(':SalesforceHybrid')";
+    var newLibDep = "compile project(':SalesforceHybrid')";
```

Comment:
We don't need `GCM` anymore because we already include it in `SalesforceSDK` for push notifications.